src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+import { SCREEN_WIDTH } from "./constants";
+
+export const isMobile = () => window.innerWidth < SCREEN_WIDTH.SM;
```

Comment:
Thank you for separating this function 👍 
Currently, all utils are stored in a separate file with the same name.
It helps to improve DX during codebase scanning. 
Would you mind following this convention, please?
Therefore, our utils implementation will be consistent.

Reply:
We can also discuss barrel file idea if you see index.ts as useful instrument to reexport all utils 👍

Reply:
I will create another MR for barrel files.

Don't want to mix that with this MR since barrelling involves changing imports in multiple places.